src/main/java/com/redhat/pantheon/servlet/AsciidocRenderingServlet.java

```diff
@@ -44,40 +43,25 @@
 import java.io.IOException;
 import java.io.Writer;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
-import java.util.stream.Collectors;
 
 import static java.util.Collections.singletonList;
 import static java.util.stream.Collectors.toList;
 
-/**
- * Hello World Servlet registered by resource type
- *
- * Annotations below are short version of:
- * 
- * @Component
- * @Service(Servlet.class)
- * @Properties({
- *    @Property(name="service.description", value="Hello World Type Servlet"),
- *    @Property(name="service.vendor", value="The Apache Software Foundation"),
- *    @Property(name="sling.servlet.resourceTypes", value="sling/servlet/default"),
- *    @Property(name="sling.servlet.selectors", value="hello"),
- *    @Property(name="sling.servlet.extensions", value="html")
- * })
- */
-@Component(
-        service = Servlet.class,
-        property = {
-                "sling.servlet.resourceTypes=pantheon/modules",
-                "sling.servlet.extensions=preview",
-                Constants.SERVICE_DESCRIPTION+"=Servlet which transforms asciidoc content into html",
```

Comment:
We lost these two properties which will be useful when keeping track of all servlets. I say we add them back.

Reply:
Which two properties? The Service Description/Vendor? I wasn't sure what they were used for - i.e., whether they populate anything in a Sling admin console anywhere - so I took them out for simplicity's sake. Do you know if the values actually get used anywhere?

Reply:
Those values are used on the OSGI management screens. For example, there is a screen where you see all the deployed components pre Bundle, and these values show up. I think it's a good documentation practice to have them, even if the code to do so is rather ugly (concatenating strings with property lines)

Reply:
Ok, I put them back.